install/update_91_92.php

```diff
@@ -296,7 +296,7 @@ function update91to92() {
       'glpi_devicesoundcardmodels',
       'glpi_devicegenericmodels',
       'glpi_devicebatterymodels',
-      'glpi_devicefirmwaremodels'
+      'glpi_devicefirmwaremodels',
```

Comment:
because I previously planned to add models of simcards. This was finally not relevant, but I left the comma here. OK to remove it